### PR TITLE
ethportal-peertest: Add scenario to test beacon bridge

### LINKED ISSUE #848: portal-bridge: Add `Test` beacon bridge mode and peertest scenario
Add `Test` beacon bridge mode and peertest scenario for testing the beacon bridge.

### PR DESCRIPTION
**Pull request by ogenev**
### What was wrong?

Closes #848 

### How was it fixed?

- Add `Test` node mode for `BeaconBridge` and add `ethportal-peertest` scenario.
- Fix a `PossibleBeaconContentValue` deserialization bug found via this test.
- Add unit tests so we can make sure this bug doesn't occur again.

### To-Do

[//]: # (Stay ahead of things, add list items here!)
- [ ] Add entry to the [release notes](https://github.com/ethereum/trin/blob/master/newsfragments/README.md) (may forgo for trivial changes)
- [x] Clean up commit history

### CODE REVIEW

File: ethportal-peertest/src/constants.rs [hunk 1]
```diff
@@ -62,3 +72,10 @@ pub const RECEIPTS_CONTENT_KEY: &str =
...
+
+// -----------------------  Beacon Content -------------------------
+
+// LighCLientUPdatesByRange
```
Comment (KolbyML) [inline ethportal-peertest/src/constants.rs:78]:
```suggestion
// LightClientUpdatesByRange
```

**Review by KolbyML — APPROVED** [review]
Looks good. I wonder if using acrynoms like LC should be used as new contributors probably wouldn't know what it means.

**Review by carver — APPROVED** [review]
Looks good!

👍🏻 for the json roundtrip test of the Beacon content

I would be pretty disappointed to have the test data blocks committed in twice. Not sure exactly what the right place for them to live is (yaml feels right, but I'm not attached to it).

File: ethportal-peertest/src/constants.rs [hunk 2]
```diff
@@ -62,3 +72,10 @@ pub const RECEIPTS_CONTENT_KEY: &str =
...
+//https://github.com/ethereum/portal-network-specs/blob/c5f0a837e7ddf2a23956c4a71b2ed136256b095e/beacon-chain/beacon-network-test-vectors.md#lightclientupdatesbyrange
+pub const LC_UPDATES_BY_RANGE_KEY: &str = "0x0130030000000000000400000000000000";
```
[the remark this answers is not in the document]
Comment (ogenev) [reply ethportal-peertest/src/constants.rs]: I was following the current pattern used in ethportal-peertest but it is obviously bad to load the same data from different places and store big hex strings as constants.

I changed this to load from the same yaml file and created an issue to move all peertest hexstr constants to yaml files https://github.com/ethereum/trin/issues/866.

**Pull request merged**